board/common/overlay/sbin/fwupdate

```diff
@@ -8,6 +8,10 @@ function exit_usage() {
     echo "           shows the current version"
     echo "       fwupdate latest"
     echo "           shows the latest available version with details"
+if [[ -n ${OS_FIRMWARE_VERSIONS_STABLE} && -n {OS_FIRMWARE_VERSIONS_BETA} ]]; then
```

Comment:
For the sake of consistency, i'd prefer to always have this option displayed (just like for the "latest" option), even when corresponding variables are not set.

I think it's better to have a simple, unified help text and to do the check in the corresponding subcommand function. I know this doesn't currently happen if `OS_FIRMWARE_LATEST_STABLE` is, for example, unset.